### DISCUSSION TITLE
OSError: [Errno 36] File name too long with option --single-notes

### DISCUSSION THREAD

**Discussion by SprickW** [vzhd1701/evernote-backup#24] [Ideas]
As I use the Evernote web clipper intensively, very long titles arise, that with "export --single-notes" result in "OSError: [Errno 36] File name too long".

I would like to see automatic shortening of the filename, possibly ending with some hash at the file names end.

**Comment by vzhd1701** [comment]
This should be fixed now. Please check out the new version (1.9.1)

**Comment by SprickW** [comment]
Also perfect! :-)

Mentioning the export options --single-notes and --include-trash with "evernote-backup --help" would make the use of export even simplier.

**Reply by vzhd1701** [reply]
They should be mentioned in `evernote-backup export --help`
